app/session.server.ts

```diff
@@ -1,7 +1,9 @@
 import { createCookieSessionStorage, redirect } from "@remix-run/node";
 import invariant from "tiny-invariant";
 
+// @ts-ignore this file no longer exists, but we might reintroduce it later
 import type { User } from "~/models/user.server";
+// @ts-ignore this file no longer exists, but we might reintroduce it later
 import { getUserById } from "~/models/user.server";
```

Comment:
I'm honestly fine if we remove all of this stuff. We can always add it back one we have a data-story

Reply:
Yup let’s just delete these unused lines.